brut.j.util/src/main/java/brut/util/BrutIO.java

```diff
@@ -30,10 +30,8 @@ public static void copyAndClose(InputStream in, OutputStream out)
         try {
             IOUtils.copy(in, out);
         } finally {
-            try {
```

Comment:
This code means is closed in and our stream, and ignore the error. But if `in.close` throws Exception affect `out.close`.`IOUtils.closeQuietly` purpose is the same, simplifying the code